.github/workflows/e2e-tests-scheduled.yaml

```diff
@@ -69,11 +69,16 @@ jobs:
         # - 'platform:el8'
         - 'ubuntu:18.04'
         - 'ubuntu:20.04'
+        - 'ubuntu:22.04'
         test_name:
         - 'manual-symmetric-key'
         - 'manual-x509'
         - 'dps-symmetric-key'
         - 'dps-x509'
+        exclude:
+          # The scheduled E2E test in the release/1.x branch would fail until the changes are merged. Excluding for now.
+          - os: 'ubuntu:22.04'
+            branch: 'release/1.4'
```

Comment:
Should this be removed?

Reply:
Yes, I think so.  On a related note, I expect there are two of the i-i-s E2E tests that will consistently fail on 22.04 that involve the keys engine.

Reply:
Removed.

> On a related note, I expect there are two of the i-i-s E2E tests that will consistently fail on 22.04 that involve the keys engine.

I assume there's no action for me to take here?